src/calculateRentalCost.js

```diff
@@ -3,8 +3,30 @@
  *
  * @return {number}
  */
+const costRental = 40;
+const mediumTerm = 3;
+const mediumTermDiscound = 20;
```

Comment:
There is a typo in the variable name `mediumTermDiscound`. It should be `mediumTermDiscount` to match the naming convention and avoid confusion.